examples/url_redacted/cmd.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/mmatczuk/anyflag"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func newCommand() *cobra.Command {
+	var (
+		site  *url.URL
+		realm []*url.URL
+	)
+
+	cmd := &cobra.Command{
+		Use: "url - print configuration at the start",
```

Comment:
Please move the example to redact and rename that. This is redact demo not an url demo.